app/components/display/search_result_component.rb

```diff
@@ -53,20 +57,30 @@ def mappings_button
 
   def visualize_button
       link_to_modal(nil, "/ajax/biomixer/?ontology=#{@ontology_acronym}&conceptid=#{@uri}", data: { show_modal_title_value: @title, show_modal_size_value: 'modal-xl' }) do
-        content_tag(:div, class: 'button') do
+        content_tag(:div, class: 'button', style: @portal_color ? "background-color: #{@portal_light_color} !important" : '') do
           concat inline_svg_tag('icons/visualize.svg')
-          concat content_tag(:div, class: 'text') { t('search.result_component.visualize') }
+          concat content_tag(:div, class: 'text', style: @portal_color ? "color: #{@portal_color} !important" : '') { t('search.result_component.visualize') }
         end
       end
   end
 
   def reveal_ontologies_button(text,id,icon)
-    content_tag(:div, class: 'button icon-right', 'data-action': "click->reveal-component#toggle", 'data-id': id) do
+    content_tag(:div, class: 'button icon-right', 'data-action': "click->reveal-component#toggle", 'data-id': id, style: @portal_color ? "background-color: #{@portal_light_color} !important" : '') do
       inline_svg_tag(icon) +
-      content_tag(:div, class: 'text') do
+      content_tag(:div, class: 'text', style: @portal_color ? "color: #{@portal_color} !important" : '') do
         text
       end +
       inline_svg_tag("icons/arrow-down.svg")
     end
   end
+
+  def portal_button(name: nil , color: nil , light_color: nil)
```

Comment:
this need to be the same as the browse page, extract this as a helper component that you use in both places.

Reply:
Not yet, I'll do it rn

Reply:
I fixed the style of it (height),
but I didn't use it in the browse cause another logic is used there.
Anyway now the style is the same.